surface-model/surfacemodel_ontology.ttl

```diff
@@ -0,0 +1,17 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix gax-core: <https://w3id.org/gaia-x/core#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix surfacemodel: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/surfacemodel/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+surfacemodel: a owl:Ontology ;
+    rdfs:label "ontology definition for surfaceModel"@en ;
+    dcterms:contributor "Sebastian Tuttas (3DM)" ;
+    owl:versionInfo "0.1"^^xsd:float .
+
+surfacemodel:surfaceModel a owl:Class ;
+    rdfs:label "class definition for surfaceModel" ;
```

Comment:
i would appreciate to have similiar terms for surfaceModel. 
See line below it is called: "surface model".

Reply:
fixed

Reply:
Now we have `surface-model` vs. `surface model` in label and comment. 
I would prefer to have one term.

as 2nd (I missed this yesterday): 
ClassNames should be PascalCase
`surfacemodel:surfaceModel` --> `surfacemodel:SurfaceModel`

Reply:
use surface-model is comment